examples/using-guidebot-flags.md

```diff
@@ -0,0 +1,106 @@
+# Using Switch Cases with Guidebot
+
+This page is an explanation of how to use switch cases with a feature in Guidebot, `message.flags`.
+
+## An Example
+
+Switch cases are a nice feature that can be used without the `message.flags` feature, and can be used like this. 
+
+```js
+// This will look for the first argument. If your command was 'hi', and you did 'hi send', it would send 'Hi!' to the channel.
+const trigger = args[0];
+
+switch (args[0]) {  
+  case 'send': {
+    // This will send in the channel that the command was run in.
+    message.channel.send('Hi!');
+    break;
+  }
+
+  case 'dm': {
+    // This will DM the message author..
+    message.author.send('Hi!');
+  }
+}
+```
+
+However, you can use the switch case with `message.flags`.
+
+```js
+  // This will look for the first argument beginning with -, the flag. If your command was 'hi', and you did 'hi -send', it would send 'Hi!' to the channel.
+switch (message.flags[0]) {
+  case 'send': {
+    // This will send in the channel that the command was run in.
+    message.channel.send('Hi!');
+    break;
+  }
+
+  case 'dm': {
+    // This will DM the message author.
+    message.author.send('Hi!');
+  }
+}
+```
+
+## Using with Arguments
+
+Let's say you're trying to make a command to add or take a role to/from a user. This can be consolidated into one command using `message.flags`. This command is based off of the [Guidebot](https://github.com/AnIdiotsGuide/guidebot) framework, and is called role.
+
+```js
+exports.run = async(client, message, args, level) => { // eslint-disable-line no-unused-vars 
+  switch (message.flags[0]) {
+    // This would be 'role -add'.
+    case 'add': {
+      // Check if the message mentions a user.
+      if (message.mentions.users.size === 0) return message.reply('Please mention a user to give the role to.');
```

Comment:
Should check for `MessageMentions#members#size`. Same for the duplicated code below.